src/cinder/audio/cocoa/DeviceManagerAudioSession.mm

```diff
@@ -264,18 +264,21 @@ @implementation AudioSessionNotificationHandlerImpl
 
 - (void)handleInterruption:(NSNotification *)notification
 {
-	NSUInteger interruptionType = [[notification.userInfo valueForKey:AVAudioSessionInterruptionTypeKey] unsignedIntValue];
+	NSUInteger interruptionType = [[notification.userInfo objectForKey:AVAudioSessionInterruptionTypeKey] unsignedIntValue];
 
 	if( interruptionType == AVAudioSessionInterruptionTypeBegan ) {
 		mDeviceManager->beginInterruption();
 	}
 	else if( interruptionType == AVAudioSessionInterruptionTypeEnded ) {
-		NSUInteger interruptionOption = [[notification.userInfo valueForKey:AVAudioSessionInterruptionOptionKey] unsignedIntValue];
-		if( interruptionOption == AVAudioSessionInterruptionOptionShouldResume ) {
-			mDeviceManager->endInterruption(true);
-		} else {
-			mDeviceManager->endInterruption(false);
-		}
+		UIApplication *app = [UIApplication sharedApplication];
+		AVAudioSession *session = [AVAudioSession sharedInstance];
+
+		auto resumeImmediately = app.applicationState == UIApplicationStateActive ||
```

Comment:
Would this be necessary if `DeviceManagerAudioSession` was registered for the `signalDidBecomeActive`?  If it still is, doesn't seem like a huge deal but perhaps it is avoidable that way.